common/build-helper/rust.sh

```diff
@@ -4,6 +4,10 @@
 export CARGO_BUILD_JOBS="$XBPS_MAKEJOBS"
 export CARGO_HOME="/host/cargo"
 
+if [ "$XBPS_SCCACHE" = "yes" ]; then
```

Comment:
I don't think we ever test for "yes", this should be `[ -n "$XBPS_SCCACHE" ]`.

Reply:
Fixed